bundle/artifacts/whl/from_libraries.go

```diff
@@ -27,7 +27,7 @@ func (*fromLibraries) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnost
 		return nil
 	}
 
-	tasks := libraries.FindAllWheelTasksWithLocalLibraries(b)
+	tasks := libraries.FindTasksWithLocalLibraries(b)
```

Comment:
Unrelated, but just curious:
 `FindTasksWithLocalLibraries` returns both, tasks that refer to a local wheel or jar file. However the code below only creates artifacts blocks for wheel files. Is this by design? Wouldn't `lib.Whl` be an empty string in those cases?